keel-ec2-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/ec2/ClusterDependencyContainer.kt

```diff
@@ -0,0 +1,74 @@
+package com.netflix.spinnaker.keel.api.ec2
+
+import com.netflix.spinnaker.keel.api.Locatable
+import com.netflix.spinnaker.keel.api.ResourceSpec
+import com.netflix.spinnaker.keel.api.SubnetAwareLocations
+import com.netflix.spinnaker.keel.core.api.ClusterDependencies
+
+/**
+ * A [ResourceSpec] that has default and override cluster dependencies (i.e. security groups, load
+ * balancers, and/or target groups).
+ */
+interface OverrideableClusterDependencyContainer<T : ClusterDependencyContainer> :
```

Comment:
Please help me think of a better name for this

Reply:
I think this is fine.

Reply:
:shrug: